tests/unit_tests/main.cpp

```diff
@@ -0,0 +1,10 @@
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+int main(int argc, char** argv) {
```

Comment:
Did you change anything that now requires us to now have a main function for the tests?

Reply:
Ah, my bad, it looks like my local git repo got slightly messed up. I meant to delete that file. I'm also missing a few other changes. Once I sort it out, I'll push a few more commits to fix it.